web/packages/teleport/src/Assist/context/AssistContext.tsx

```diff
@@ -550,6 +550,10 @@ export function AssistContextProvider(props: PropsWithChildren<unknown>) {
 
   useEffect(() => {
     loadConversations();
+
+    return () => {
+      window.clearTimeout(refreshWebSocketTimeout.current);
```

Comment:
Can you add that this can be removed after https://github.com/gravitational/teleport.e/pull/1609 is implemented?